themes/10up-theme/tailwind.config.js

```diff
@@ -81,6 +93,7 @@ module.exports = {
 		/*
 			Add prose classes to provide sensible typography styles to longform content blocks
 			See: https://tailwindcss.com/docs/plugins#typography
+			Comment out this plugin to add project specific typography styles
```

Comment:
@Nikki-Jones I feel like this comment is kind of misleading. It reads as though you either have to use this plugin, or custom typography styles. 

The main thing this plugin does is just add the `.prose-*` classes, which have to be specifically applied to be used. The other typography configuration is handled entirely in the config, so I feel like someone who doesn't know Tailwind would be very easily confused by this. 

```suggestion
			You may remove this plugin if you're not using the `.prose-*` classes for long-form content.
```